docker-compose.yml

```diff
@@ -214,12 +214,16 @@ services:
       - "8126:8126"
 
   # https://hub.docker.com/_/elasticsearch/
-  # Note: This image is deprecated, but the new one requires fiddling.
-  elasticsearch:
+  legacy-elasticsearch:
     image: mozilla/socorro_elasticsearch:1.4.5
     ports:
       - "9200:9200"
-      - "9300:9300"
+
+  # https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html
```

Comment:
Eventually `current` won't point to 8.14.3 anymore, right? If we're going to fix the version we're upgrading to, can this link point to that specific version as well?